### PR TITLE
terraform complete for s3 bucket, tested using boto3

### PR DESCRIPTION
**Pull request by lukieh2014**
Completed terraform for S3 bucket (for storing daily summary PDF reports). Tested and working using boto3. Referring to ticket #50 .

### CODE REVIEW

File: terraform/s3.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+provider "aws" {
+  region = "eu-west-2"
+}
+
+resource "aws_s3_bucket" "c14-bandcamp-reports" {
+  bucket = "c14-bandcamp-reports"
+  acl    = "private"
```
Comment (bentrzcinski) [inline terraform/s3.tf:7]: Does this mean the S3 is private? Will that work with everything else being public?

File: terraform/s3.tf [hunk 2]
```diff
@@ -0,0 +1,13 @@
+provider "aws" {
+  region = "eu-west-2"
+}
+
+resource "aws_s3_bucket" "c14-bandcamp-reports" {
```
[the remark this answers is not in the document]
Comment (lukieh2014) [reply terraform/s3.tf:5]: yeah good point, are we going with "c14-bandcamp-...." then ?

File: terraform/s3.tf (hunk 1 above)
Comment (lukieh2014) [reply terraform/s3.tf:7]: yes I tested with boto3 and was able to download a file from the s3 bucket.

File: terraform/s3.tf [hunk 3]
```diff
@@ -0,0 +1,13 @@
+provider "aws" {
+  region = "eu-west-2"
+}
```
[the remark this answers is not in the document]
Comment (lukieh2014) [reply terraform/s3.tf]: cool, just removed it!

File: terraform/s3.tf (hunk 1 above)
Comment (lukieh2014) [reply terraform/s3.tf:4]: just tested with uploading a file and that worked too

File: terraform/s3.tf (hunk 2 above)
[the remark this answers is not in the document]
Comment (bentrzcinski) [reply terraform/s3.tf:2]: Yeah that sounds good

**Review by bentrzcinski — APPROVED** [review]
Good stuff

**Review by emily-curtis — APPROVED** [review]
approved!

**Review by ebradley12 — APPROVED** [review]
Looks good

**Pull request merged**